modules/ROOT/pages/connected-app-bearer-token-example.adoc

```diff
@@ -0,0 +1,64 @@
+= Getting the Bearer Token for a Connected App Example
+
+This example demonstrates how to create a connected app and get the bearer token so that you can invoke Anypoint Platform APIs. 
```

Comment:
Should we say specifically that this example is for a connected app that acts on its own behalf (client credentials)?